saleor/graphql/schema.graphql

```diff
@@ -2916,6 +2913,9 @@ type Order implements Node & ObjectWithMetadata {
   totalBalance: Money!
   userEmail: String
   isShippingRequired: Boolean!
+  discount: Money @deprecated(reason: "Use discounts field. This field will be removed after 2021-08-01")
+  discountName: String @deprecated(reason: "Use discounts field. This field will be removed after 2021-08-01")
+  translatedDiscountName: String @deprecated(reason: "Use discounts field. This field will be removed after 2021-08-01")
```

Comment:
Yes, we should.

Reply:
AFAIK we decided some time ago to use date instead of the version. (User already know that he has 6 months to add some changes in his code. With Saleor4.0 I don't know the ETA when this will be removed.)

Reply:
The problem is that we are not sure about any major release date, maybe this will be improved in the future when the release cycle is better organized. For now, it's safer to say Saleor 4.0 - which is the next version of Saleor where we will be able to remove any public field from the schema.